src/lib/realtime/index.tsx

```diff
@@ -46,15 +46,15 @@ export function Provider({ children, client: passedClient }: { children: ReactNo
                 userQuery.status === 'error' ||
                 (userQuery.status === 'success' && !userQuery.state.data?.currentUser?.id)
             ) {
-                setShouldRefetchCurrentUser(true);
+                // setShouldRefetchCurrentUser(true);
             }
         }
     }, [currentUser, userQuery.state.data?.currentUser, userQuery.status]);
 
     // Check every 5 seconds if we should refetch the RealtimeCurrentUser query
     useInterval(() => {
         if (shouldRefetchCurrentUser) {
-            userQuery.refetch({ requestPolicy: 'cache-and-network' });
```

Comment:
Did you mean to leave this commented out?

Reply:
lol nope